docs/static/js/scripts.js

```diff
@@ -18,35 +18,20 @@ var links = document.querySelectorAll('a');
 links.forEach(function(link) {
   var parentClass = link.parentNode.classList;
   var ignoreNav = !(parentClass.contains('p-navigation__logo') || parentClass.contains('p-navigation__link'));
-  if (link.hostname && link.hostname != location.hostname && ignoreNav) {
+
+  var isInternal = link.href.indexOf('https://docs.vanillaframework.io') === 0 || link.href.indexOf('https://vanillaframework.io') === 0;
+
+  if (!isInternal && link.hostname && link.hostname != location.hostname && ignoreNav) {
     link.className += ' p-link--external';
   }
 });
 
 // Docs search functions
-function resetFilter() {
-  document.getElementById('docs-list-sorted').classList.add('u-hide');
-  document.getElementById('docs-list-unsorted').classList.remove('u-hide');
-}
-
-function filterDocs() {
-  var input = document.getElementById('search-docs');
-  var filter = input.value.toLowerCase();
+var searchDocsReset = document.getElementById('search-docs-reset');
+var searchBox = document.getElementById('search-docs');
 
-  if (filter) {
-    document.getElementById('docs-list-unsorted').classList.add('u-hide');
-    document.getElementById('docs-list-sorted').classList.remove('u-hide');
-
-    var docsListItems = document.querySelectorAll('#docs-list-sorted .p-sidebar-nav__item');
-    docsListItems.forEach(function(item) {
-      var text = item.firstElementChild.innerText.toLowerCase();
-      if (text.indexOf(filter) === 0) {
-        item.classList.remove('u-hide');
-      } else {
-        item.classList.add('u-hide');
-      }
-    });
-  } else {
-    resetFilter();
-  }
-}
+searchDocsReset.addEventListener('click', function(e) {
```

Comment:
Didn't a type reset in a form do this without js?

Reply:
It doesn't on input type="search" 🤷‍♂ https://jsfiddle.net/bartaz/L1a9npse/
I guess because browsers have custom 'clear' functionality for search that we hide.

Reply:
Oh, it's actually something different: https://jsfiddle.net/bartaz/L1a9npse/

"Reset" button does work (both on text and search inputs) but it doesn't clear them, it sets them to whatever was set as value in HTML. So if we set value in HTML (what we do with search input on search page) it will not clear it, just reset to original value.

Reply:
Oh yea, I remember now. Thanks